docs/supportedsites.md

```diff
@@ -226,6 +226,7 @@
  - **Instagram**
  - **instagram:user**: Instagram user profile
  - **InternetVideoArchive**
+ - **IOL**: iol.pt sites (tvi archive/stream and maisfutebol)
```

Comment:
Fixed.

Thanks for the comment.

I usually don't think files in the repo are auto-generated so I took the liberty.